File: API/Models/Entities/MorningChecklist.cs
namespace API.Models.Entities {
    public class MorningChecklist : Checklist {
        public int MorningChecklistID { get; set; }
        public override int UserID { get; set; }
        public override DateOnly Date { get; set; }
        public bool? GlassOfWater { get; set; }
        public bool? Meds { get; set; }
        public bool? Vitamins { get; set; }
        public bool? Breakfast { get; set; }
        // public override int GetID() { return MorningChecklistID; }
    }
}

File: client/src/app/helpers/models/morningChecklist.ts

export interface MorningChecklistInterface {
    date: Date;
    glassOfWater: boolean;
    meds: boolean;
    vitamins: boolean;
    breakfast: boolean;
}

export class MorningChecklist {
    date = new Date();
    glassOfWater = false;
    meds = false;
    vitamins = false;
    breakfast = false;

    constructor() {
    }
}
